cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -64,6 +67,60 @@ def incremental_backup(volume_backup, provider):
         logger.warning('Exception during volume backup deletion - skipping..')
 
 
+@pytest.fixture(scope='function')
+def new_instance(provider):
+    prov_data = provider.data['provisioning']
+    prov_form_data = {
```

Comment:
I think its not proper way to implement it. Can you please explain why you are not pulling it from yaml file?

Reply:
Could you clarify please

I just looked on existing examples and  it is implemented like that
Also isn't provider.data['provisioning'] fetched from yaml files ?

Reply:
Since it's optional, please approve changes for now.
Anyway i want to try it offline but still not getting your point.

template_name here is referring image name, cirros in my case which is not mentioned in the test.